toolkit/offchain/tests/integration_tests.rs

```diff
@@ -92,9 +93,25 @@ async fn governance_flow() {
 	let client = initialize(&container).await;
 	let genesis_utxo = run_init_goveranance(&client).await;
 	let _ = run_update_goveranance(&client, genesis_utxo).await;
-	assert!(run_upsert_d_param(genesis_utxo, 0, 1, &eve_payment_key(), &client)
-		.await
-		.is_some());
+	let await_tx = FixedDelayRetries::new(Duration::from_millis(500), 100);
+	// TODO: ETCM-9576 run upsert for permissioned candidates and then d-param and sign and submit both later, to prove they can interleve
+	let upsert_result = permissioned_candidates::upsert_permissioned_candidates(
+		genesis_utxo,
+		&vec![],
+		&governance_authority_payment_key(),
+		&client,
+		&await_tx,
+	)
+	.await
+	.unwrap();
+	if let Some(McSmartContractResult::TxCBOR(tx_cbor)) = upsert_result {
+		let result =
+			run_assemble_and_sign(&tx_cbor, &[EVE_PAYMENT_KEY, GOVERNANCE_AUTHORITY_KEY], &client)
```

Comment:
For now `upsert_permissioned_candidates` and `update_governance` don't sign transaction if real multisig is used. I think we should modify them to add the first signature. It makes it simpler and also makes a statement, that author is really committed to spending money on tx.